extension/chrome/src/js/storage.js

```diff
@@ -27,196 +27,247 @@ var extensionStorage = (function() {
 		checkversion: true
 	};
 
-	var cloudStorage = chrome.storage.sync;
+	let cloudStorage = chrome.storage.sync;
```

Comment:
[not related to this PR but anyway] is this line executed before or after `chrome = chrome || browser` in `background.js`?

because `browser.storage.sync` is available in FF 53+ https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage/sync